src/app/components/Button.svelte

```diff
@@ -22,6 +22,13 @@
     flex-direction: column;
     align-items: center;
   }
+
+  @media (max-width: 425px) {
```

Comment:
Do we want to create a CSS variable for `425px` in `global.css`, like how we have variables for our CSS colours? Same question applies for the other sizes

Reply:
Ohh I thought this was such a great idea, but it seems like `var()` doesn't work on media queries ([related StackOverflow thread](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries)). Should we create a .ts file to store constants?

Reply:
Oh, that's too bad :(
The link you provided was very insightful.

I think a .ts file could definitely work for our purposes! The industry standard though would be to use a CSS preprocessor like PostCSS or SCSS, which I _think_ compiles CSS for us, which can circumvent the problems we are having currently. The counterargument might be that either of these is overkill currently. What are your thoughts on this?

I'm totally fine if we want to mark this a resolved, and then create an investigation task about the best way to go about these types of cases (and talk about pros and cons about `.ts` files vs `.scss` files)

Reply:
Will do! Thanks for looking into it, I'll open up a separate ticket to investigate further into this.